examples/create_letters_from_csv/letter.py

```diff
@@ -0,0 +1,148 @@
+# usage:
+#   python letter.py input.csv
+#   logs to csv's in the output directory
+
+import csv
+import datetime
+import lob
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath(__file__+'../../..'))
+
+###########################################################
+# TODO: Provide the following constants
```

Comment:
See comment below, but do you think that will be sufficient? Or should the TODO be removed as well? I like it because it will sometimes be highlighted in the users editor to alert them to something they need to do.